crates/control/src/motion/jump_left.rs

```diff
@@ -6,26 +6,29 @@ use motionfile::{MotionFile, MotionInterpolator};
 use types::{
     condition_input::ConditionInput,
     cycle_time::CycleTime,
-    joints::{Joints, JointsCommand},
+    joints::JointsCommand,
     motion_selection::{MotionSafeExits, MotionSelection, MotionType},
+    sensor_data::SensorData,
 };
 
 pub struct JumpLeft {
-    interpolator: MotionInterpolator<Joints<f32>>,
+    interpolator: MotionInterpolator<JointsCommand<f32>>,
 }
 
 #[context]
 pub struct CreationContext {
-    hardware_interface: HardwareInterface,
+    pub hardware_interface: HardwareInterface,
+    pub motion_safe_exits: PersistentState<MotionSafeExits, "motion_safe_exits">,
 }
 
 #[context]
 pub struct CycleContext {
-    motion_safe_exits: PersistentState<MotionSafeExits, "motion_safe_exits">,
+    pub motion_safe_exits: PersistentState<MotionSafeExits, "motion_safe_exits">,
 
-    condition_input: Input<ConditionInput, "condition_input">,
-    cycle_time: Input<CycleTime, "cycle_time">,
-    motion_selection: Input<MotionSelection, "motion_selection">,
+    pub condition_input: Input<ConditionInput, "condition_input">,
+    pub cycle_time: Input<CycleTime, "cycle_time">,
+    pub motion_selection: Input<MotionSelection, "motion_selection">,
+    pub sensor_data: Input<SensorData, "sensor_data">,
```

Comment:
These `pub`s are legacy. You can remove them

Reply:
Done